completions/conda.completion.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
-command -v register-python-argcomplete &> /dev/null \
+_omb_util_command_exists register-python-argcomplete &> /dev/null \
```

Comment:
```suggestion
_omb_util_command_exists register-python-argcomplete \
```

Now we can remove all `&>/dev/null` with `_omb_util_command_exists` because that redirection is now included in the function `_omb_util_command_exists` itself.

There are many similar cases.